server/src-lib/Hasura/GraphQL/Resolve/Context.hs

```diff
@@ -1,24 +1,14 @@
 module Hasura.GraphQL.Resolve.Context
-  ( FieldMap
-  , RelationInfoMap
-  , FuncArgItem(..)
-  , OrdByCtx
-  , OrdByItemMap
+  ( FuncArgItem(..)
   , OrdByItem(..)
-  , FuncArgSeq
-  , PGColArgMap
   , UpdPermForIns(..)
   , InsCtx(..)
-  , InsCtxMap
```

Comment:
All of these are now part of `Resolve.Types` module and that is exposed by this module.